### ISSUE TITLE
CRX_HEADER_INVALID

### ISSUE THREAD

**Issue by skiddykong** [Capgemini/dtx-polyfiller#3]
Chrome version:
Version 63.0.3239.132 (Official Build) (64-bit)

When I drag the file as per the instructions I get the error:

Package is invalid: CRX_HEADER_INVALID

**Comment by Ishino** [comment]
this might be caused by a corrupted download ( virus scanners etc. )

Try building the extension 

```
    Install Node/Yarn
    Run yarn
    Populate .env based on example.env template
    Run yarn build
```

**Comment by daniel-cotton** [comment]
Addressed via Chrome Web Store distribution in v2.0.2
